app/views/producer_mailer/order_cycle_report.html.haml

```diff
@@ -37,8 +38,9 @@
         %tr
           %td
             #{line_items.first.variant.sku}
-          %td
-            #{raw(line_items.first.product.supplier.name)}
+          - if @distributors_pickup_times.many?
+            %td
+              #{raw(line_items.first.product.supplier.name)}
```

Comment:
Not related to this PR but I'm wondering why the name is raw here. It looks like it got copied from the text template without knowing that it's introducing a security issue. A supplier could inject HTML in the name.

* https://github.com/openfoodfoundation/openfoodnetwork/commit/6ecf896fa2b34509f5d915e4d8a92d957892ac90#diff-8811f57637800c39d8e1e157c4d1fca015d5d0c9ed4fb8eb277210315396e797

Reply:
This will be addressed here #11801